component/discovery/hetzner/hetzner.go

```diff
@@ -0,0 +1,77 @@
+package hetzner
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/component/common/config"
+	"github.com/grafana/agent/component/discovery"
+	"github.com/prometheus/common/model"
+	prom_discovery "github.com/prometheus/prometheus/discovery/hetzner"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:    "discovery.hetzner",
+		Args:    Arguments{},
+		Exports: discovery.Exports{},
+
+		Build: func(opts component.Options, args component.Arguments) (component.Component, error) {
+			return New(opts, args.(Arguments))
+		},
+	})
+}
+
+type Arguments struct {
+	Role             string                  `river:"role,attr"`
+	RefreshInterval  time.Duration           `river:"refresh_interval,attr,optional"`
+	Port             int                     `river:"port,attr,optional"`
+	HTTPClientConfig config.HTTPClientConfig `river:",squash"`
+}
+
+var DefaultArguments = Arguments{
+	Port:             80,
+	RefreshInterval:  time.Duration(60),
```

Comment:
```suggestion
	RefreshInterval:  time.Duration(60 * time.Second),
```

Otherwise, I think it will be 60ns

Reply:
Are you sure? If I leave `* time.Second` the linter complaints of redundant formating

Reply:
Oops, you're right, because 60 * time.Second will already be a duration; I've amended the suggestion above.

The [time.Duration](https://pkg.go.dev/time#Duration) type is an alias for int64, signifying nanoseconds. The current code would simply convert the 60 int to ns.